.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: php
+
 php:
-    - 5.5
-    - 5.4
+    - 7.2
+    - 7.3
+    - 7.4
 
-before_script:
-    - composer selfupdate
-    - composer install --prefer-source
+env:
+    matrix:
+        - COMPOSER_FLAGS="--prefer-lowest"
+        - COMPOSER_FLAGS=""
 
-script: phpunit
\ No newline at end of file
+before_script:
+    - travis_retry composer self-update
+    - travis_retry composer update ${COMPOSER_FLAGS} --no-interaction --prefer-source
```

Comment:
Why doesn't it run the tests anymore and why does it run `update` instead of `install`? (Bear with me a bit, I've been outside the PHP eco system for a couple of years)

Reply:
Your composer lock was using very old dependencies. I find it a good idea to test new package updates with the newest dependencies. It shows a more current state of the package.